app/main.py

```
@@ -138,120 +169,136 @@ async def bot_connect(
-    # 3. Generate unique session ID and client session ID for this subprocess
-    session_id = str(uuid.uuid4())  # Always generate random session ID
-    client_sid = request.sessionId or str(
-        uuid.uuid4()
-    )  # Use client-provided sessionId or generate fallback
-    logger.bind(session_id=session_id).info(
-        f"Generated session ID for new voice agent: {session_id}"
-    )
-    logger.bind(client_sid=client_sid).info(
-        f"Using client session ID for new voice agent: {client_sid}"
-    )
+        return {"room_url": room_url, "token": token, "session_id": session_id}
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)